src/remotestorage.js

```diff
@@ -447,9 +447,9 @@
       this.backend = what;
       if (this.localStorageAvailable()) {
         if (what) {
-          localStorage['remotestorage:backend'] = what;
+          localStorage.setItem('remotestorage:backend', what);
```

Comment:
Question unrelated to these changes: if it doesn't set a backend at all (when local storage is not available), then how does it know which backend to use later on?